pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -273,6 +273,20 @@ func (r *ReconcileComplianceSuite) reconcileRemediations(suite *compv1alpha1.Com
 		return err
 	}
 
+	// refresh remediationList
+	if err := r.client.List(context.TODO(), &remList, &listOpts); err != nil {
+		return err
+	}
+
+	// Check that all remediations have been applied yet. If not, cause an error
+	// and requeue.
+	for _, rem := range remList.Items {
+		if rem.Status.ApplicationState != compv1alpha1.RemediationApplied {
+			return fmt.Errorf("Remediation '%s' not applied yet", rem.Name)
```

Comment:
Wait, isn't this always going to match unless the remediations are auto-applied?

Reply:
you're right

Reply:
Done